<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_ingresarProceso">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>271</width>
          <height>211</height>
         </rect>
        </property>
        <property name="title">
         <string>Ingresar Proceso</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>26</y>
           <width>119</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Nombre del Programador</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>55</y>
           <width>49</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Operaci贸n</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>84</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Primer Dato</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>113</y>
           <width>68</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Segundo Dato</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>142</y>
           <width>73</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Tiempo maximo</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_nombreProgramador">
         <property name="geometry">
          <rect>
           <x>135</x>
           <y>26</y>
           <width>122</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_agregarProceso">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>171</y>
           <width>251</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Agregar</string>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBox_operacion">
         <property name="geometry">
          <rect>
           <x>135</x>
           <y>55</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>+</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>-</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>*</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>/</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>%</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>^</string>
          </property>
         </item>
        </widget>
        <widget class="QSpinBox" name="spinBox_primerDato">
         <property name="geometry">
          <rect>
           <x>135</x>
           <y>84</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_segundoDato">
         <property name="geometry">
          <rect>
           <x>135</x>
           <y>113</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_Tiempo">
         <property name="geometry">
          <rect>
           <x>135</x>
           <y>142</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Procesos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Estatus</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_lotesPendientes">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Lotes Pendientes</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="pushButton_iniciar">
             <property name="text">
              <string>Iniciar</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Contador Global</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLCDNumber" name="lcdNumber"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Proceso en ejecuci贸n</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget_procesoEjecucion">
             <property name="rowCount">
              <number>6</number>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <row>
              <property name="text">
               <string>Programa</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Nombre</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Operaci贸n</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Tiempo Estimado</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Tiempo Transcurrido</string>
              </property>
             </row>
             <row>
              <property name="text">
               <string>Tiempo Restante</string>
              </property>
             </row>
             <column/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Procesos terminados</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget_procesosTerminados">
             <property name="columnCount">
              <number>5</number>
             </property>
             <column>
              <property name="text">
               <string>Programa</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Primer Dato</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Operaci贸n</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Segundo Dato</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Resultado</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Lote en ejecuci贸n</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget_loteEjecucion">
             <property name="rowCount">
              <number>0</number>
             </property>
             <property name="columnCount">
              <number>2</number>
             </property>
             <column>
              <property name="text">
               <string>Programa</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Tiempo Estimado</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_nombreProgramador</tabstop>
  <tabstop>comboBox_operacion</tabstop>
  <tabstop>spinBox_primerDato</tabstop>
  <tabstop>spinBox_segundoDato</tabstop>
  <tabstop>spinBox_Tiempo</tabstop>
  <tabstop>pushButton_agregarProceso</tabstop>
  <tabstop>lineEdit_lotesPendientes</tabstop>
  <tabstop>tableWidget_loteEjecucion</tabstop>
  <tabstop>tableWidget_procesoEjecucion</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
